doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -4564,7 +4759,20 @@ Trusted Firmware-M (TF-M)
 
 The issues in this section are related to the TF-M implementation in the |NCS|.
 
-.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0
+.. rst-class:: v2-8-0
+
+NCSDK-29095: Writing assets to RRAM using TF-M causes increased interrupt latencies
+  Writes to RRAM provoke increased latencies to even the highest-priority interrupts.
```

Comment:
```suggestion
  Writes to RRAM provoke increased latencies even for the highest-priority interrupts.
```

Reply:
Not so sure about this. Can you @mia-ko check with the security guys?

Reply:
Approving this suggestion. :slightly_smiling_face: